pkg/reconciler/eventlistener/resources/deployment.go

```diff
@@ -37,11 +37,22 @@ const (
 )
 
 var (
-	strongerSecurityPolicy = corev1.PodSecurityContext{
+	baseStrongerSecurityPolicy = corev1.PodSecurityContext{
 		RunAsNonRoot: ptr.Bool(true),
+		SeccompProfile: &corev1.SeccompProfile{
+			Type: corev1.SeccompProfileTypeRuntimeDefault,
+		},
 	}
 )
 
+func getStrongerSecurityPolicy(cfg *config.Config) corev1.PodSecurityContext {
+	securityContext := baseStrongerSecurityPolicy
+	securityContext.RunAsUser = ptr.Int64(cfg.Defaults.DefaultRunAsUser)
+	securityContext.RunAsGroup = ptr.Int64(cfg.Defaults.DefaultRunAsGroup)
```

Comment:
Hi @kristofferchr 
For runAsUser and runAsGroup 
can you handle similar to this https://github.com/tektoncd/triggers/blob/main/pkg/reconciler/eventlistener/resources/container.go#L71-L76 
because In a platform like OpenShift we set `""` value so lets keep runAsUser and runAsGroup when its actually set

Reply:
Certainly, FSGroup as well?

Reply:
Implemented and added tests.